### PR TITLE
Amit create new check script

### PR DESCRIPTION
**Pull request by amit-chandak-unskript**
## Description
New script to create new check directory structure

### Testing
```
base) amits-mbp-2:custom-checks amit$ python3 awesome/unskript-ctl/unskript-add-check.py -t AWS -n aws_list_ebs -d "List EBS volumes"
(base) amits-mbp-2:custom-checks amit$ cat actions/aws_
aws_get_ecs_services_status_v2/ aws_list_ebs/                   aws_list_unused_secrets/
(base) amits-mbp-2:custom-checks amit$ cat actions/aws_list_ebs/
__init__.py        aws_list_ebs.json  aws_list_ebs.py
(base) amits-mbp-2:custom-checks amit$ cat actions/aws_list_ebs/aws_list_ebs.*
{
  "action_title": "aws_list_ebs",
  "action_description": "List EBS volumes",
  "action_type": "LEGO_TYPE_AWS",
  "action_entry_function": "aws_list_ebs",
  "action_needs_credential": true,
  "action_output_type": "ACTION_OUTPUT_TYPE_LIST",
  "action_is_check": true,
  "action_supports_iteration": true,
  "action_supports_poll": true
}from pydantic import BaseModel, Field
from typing import Optional, Tuple
import pprint


class InputSchema(BaseModel):
    pass

def aws_list_ebs_printer(output):
    pass

def aws_list_ebs(handle) -> Tuple:
    pass(base)
```

### CODE REVIEW

File: unskript-ctl/unskript-add-check.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            json_data['action_description'] = args.description
+            json_data['action_type'] = "LEGO_TYPE_" + args.type
+            json_data['action_entry_function'] = args.name
+            json_data['action_needs_credential'] = True
+            json_data['action_output_type'] = "ACTION_OUTPUT_TYPE_LIST"
+            json_data['action_is_check'] = True
+            json_data['action_supports_iteration'] = True
+            json_data['action_supports_poll'] = True
+        except:
+            pass
```
Comment (jayasimha-raghavan-unskript) [inline unskript-ctl/unskript-add-check.py:56]: Should we raise an error? instead of pass

File: unskript-ctl/unskript-add-check.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            file = open( custom_dir + "/" + "__init__.py","w")
+            file.close()
+        except Exception as e:
+            print(f"Unable to create __init__.py File {e}")
+
+        AWESOME_DIRECTORY = "Awesome-CloudOps-Automation"
+        environment = Environment(loader=FileSystemLoader(current_directory + "/" + AWESOME_DIRECTORY + "/unskript-ctl/templates/"))
+        template = environment.get_template("check.py.template")
+
+        content = template.render({"check_function_name": args.name})
```
Comment (jayasimha-raghavan-unskript) [inline unskript-ctl/unskript-add-check.py:78]: Arent we creating README.md?

File: unskript-ctl/templates/check.py.template [hunk 3]
```diff
@@ -0,0 +1,13 @@
+from pydantic import BaseModel, Field
+from typing import Optional, Tuple
+import pprint
+
+
+class InputSchema(BaseModel):
+    pass
```
Comment (jayasimha-raghavan-unskript) [inline unskript-ctl/templates/check.py.template:7]: Should we add some comments like what could a sample InputSchema be? like what is the format we expect, need to have a Field, need to have a title, description and/or default value. Etc..?

**Review by jayasimha-raghavan-unskript — DISMISSED** [review]
Could you please take care of Minor comments.

File: unskript-ctl/unskript-add-check.py (hunk 2 above)
Comment (amit-chandak-unskript) [reply unskript-ctl/unskript-add-check.py:78]: No, not yet

**Review by jayasimha-raghavan-unskript — APPROVED** [review]

**Pull request merged**